ReportPortal.ATM/TestProject/Features/DashboardAddWidget.feature

```diff
@@ -0,0 +1,15 @@
+﻿Feature: Dashboard Add Widget feature
+
+User is able to add widget to dashboard
+
+@Smoke
+Scenario: Add a Widget
```

Comment:
Same here, the scenario name should be just like a test case name, e.g.:

* Verify the user can add a widget to the dashboard.